docs/compilation/modules.md

```diff
@@ -0,0 +1,57 @@
+# Modules
+
+{% hint style="warning" %}
+Modules are still experimental. They are only compatible with composition, that is, all function outputs can be used as inputs for every functions. The crypto-parameters used in this mode are pretty large, and will likely give slow execution time.
+{% endhint %}
+
+In some cases, it might be interesting to deploy a server that is able to execute different functions. With *Concrete*, it is possible to compile fhe _modules_, that can contain many different functions, at once. All the functions are compiled in a single step and can be [deployed with the same artifacts](../guides/deploy.md#deployment-of-modules). Here is an example:
+
+```python
+from concrete import fhe
+
+@fhe.module()
+class MyModule:
+    @fhe.function({"x": "encrypted"})
+    def inc(x):
+        return x + 1 % 20
+
+    @fhe.function({"x": "encrypted"})
+    def dec(x):
+        return x - 1 % 20
+```
+
+The `MyModule` fhe module can then be compiled using the `compile` method, by providing a dictionnary of input sets for every functions:
```

Comment:
You can compile the FHE module `MyModule` using the `compile` method. To do that, you need to provide a dictionary of input sets for every function: